### DISCUSSION TITLE
Week 9 - Summary and Project Update

### DISCUSSION THREAD

**Discussion by bst3ck** [USU-ITLS-x265-23-summer/class-repo#179] [General]
## API Adventure
This has actually be a very interesting week. I probably spent way too much time playing with the Google APIs and I should have spent more time on my project. I actually went a little off track and spent a lot of time trying to see if I could write my own API services. I am pretty good with PHP and have a server that runs a couple of back-end jobs that I built and I wanted to see if I could write some API's that I could access for this project. It is not as easy as some of the examples. But it does give me some ideas about how I can offload some of the javascript to a backend system running PHP (which I am much more familiar with). I have not been able to successfully do this as I would have to open my server up outside my network, but it has promise.

## Project Update
As I mentioned above, I had a major squirrel moment after doing the API tutorials. I am still working on the HTML and CSS for the main page which is taking much longer then I expected. What I am having the most problems with is building the page and then realizing that once I view it on mobile, it looks terrible. So I am having to spend a lot of time double-building each element so they look good on both display sizes.

I am not sure how to share my repo, so I am just going to include the link here for the class to review. I would love any feedback anyone has or suggestions on how to make it better.

Link to Project Repo:
https://github.com/USU-ITLS-x265-23-summer/bst3ck-final-project

**Comment by derekscholes** [comment]
Do you by chance have a link to the webpage you created with this repo? If none was made let me know. I would like to be able to compare the code with the current webpage you created with it so that I don't have to copy it into my own editor. If you have questions about how to do so, let me know and I can send you some helpful videos.

**Reply by bst3ck** [reply]
I have not loaded it to my hosting server yet, but let me deploy it so you can see what I have released to my main branch.

**Reply by bst3ck** [reply]
Here is the link to my aws server where this is hosted: http://www.bksteckler.com/klokke/

**Reply by coopsand** [reply]
This looks amazing! Super super cool to see your progress! I love the look and overall design of the site.
